core-plugins/src/main/java/co/cask/hydrator/plugin/batch/action/EmailAction.java

```diff
@@ -200,7 +200,7 @@ public PasswordAuthentication getPasswordAuthentication() {
       ClassLoader oldClassLoader = Thread.currentThread().getContextClassLoader();
       Thread.currentThread().setContextClassLoader(getClass().getClassLoader());
       try {
-        Transport transport = session.getTransport(config.protocol);
+        Transport transport = session.getTransport(config.protocol.toLowerCase());
```

Comment:
I guess its unlikely that someone will use a transport with a non-lowercase protocol, since most people will just use default protocols.
LGTM